packages/dashboard/src/customization/widgets/barChart/plugin.tsx

```diff
@@ -14,6 +14,8 @@ export const barChartPlugin: DashboardPlugin = {
         icon: BarIcon,
       },
       properties: () => ({
+        aggregationType: 'AVERAGE',
```

Comment:
does bar chart have these at a root level? I thought it was still using the old query config?